include/dfu/suit_dfu_fetch_source.h

```diff
@@ -65,37 +72,38 @@ typedef int (*suit_dfu_fetch_source_request_fn)(const uint8_t *uri, size_t uri_l
 /**
  * @brief Register fetch source
  *
- * @param[in]   request_fn		Fetch source callback function. The suit processor will
- * iterate over registered callback functions when fetching from an URI is needed.
- * Attention! Order of execution of callback functions is not guaranteed!
- *
+ * @param[in] request_fn  Fetch source callback function. The suit processor will iterate over
+ *                        registered callback functions when fetching from an URI is needed.
```

Comment:
"from an URI" sounds strange, like "fetch from an identifier". It should be more like "fetch from a location pointed by the identifier". Maybe "fetching based on URI" would be short and clear enough?